src/lib/errors/request-validation-error.js

```diff
@@ -0,0 +1,16 @@
+import HttpStatus from 'http-status-codes';
+import BaseHttpError from './base-http-error';
+
+class RequestValidationError extends BaseHttpError {
+  static message = 'All fields required.';
```

Comment:
What is this needed for and why is it static?
Message should be generic something like `"RequestValidationError occurred, check your data on the client"`

Reply:
This goes for all errors

Reply:
BTW You can fix them up in a later PR